examples/apps/cchmc_ped_abd_ct_seg_app/segmentation_overlay_operator.py

```
@@ -0,0 +1,959 @@
+
+    # Define label dictionary
+    label_dict = {
+        "liver": 1,
+        "spleen": 2,
+        "kidney": 3,
+    }
+
+    # Test 1: Without GPU (CPU only)
+    print("\n[Test 1] Running with CPU (use_gpu=False)...")
+    fragment1 = Fragment()
+    operator_cpu = SegmentationMetricsOperator(fragment1, use_gpu=False)
```

Ruff (0.15.12)

[error] Undefined name `SegmentationMetricsOperator`

(F821)